libs/pages/user/src/lib/feature/page-user-general-feature/page-user-general-feature.tsx

```diff
@@ -23,7 +23,7 @@ export function PageUserGeneralFeature() {
     defaultValues: {
       firstName: user.first_name,
       lastName: user.last_name,
-      email: user.user_email,
+      email: user.user_email !== '' ? user.user_email : userToken.email!,
```

Comment:
This doesn't work for me.
This does:
`email: user.user_email ?? userToken.email`

Reply:
Thanks, I couldn't test with my account